crates/core/src/factories/core_space.rs

```diff
@@ -92,13 +134,142 @@ impl Space for CoreSpace {
 
     fn local_agent_join(
         &self,
-        _local_agent: agent::DynLocalAgent,
+        local_agent: agent::DynLocalAgent,
     ) -> BoxFut<'_, K2Result<()>> {
-        Box::pin(async move { todo!() })
+        Box::pin(async move {
+            // set some starting values
+            local_agent.set_tgt_storage_arc_hint(DhtArc::Empty);
```

Comment:
This is fine for now, I'd like to come back to this per the comments I've left above